src/labels/CustomLabels.labels

```diff
@@ -305,36 +353,36 @@
+        <shortDescription>stgHelpRelAutoCreatedDup</shortDescription>
+        <value>Auto-created Relationships are auto-deduplicated to prevent duplicate Relationship records from being created between two Contacts. Check this box to disable this behavior &amp; allow multiple Relationships of the same Type to be created between two Contacts</value>
     </labels>
     <labels>
-        <fullName>stgHelpStoreErrorsOn</fullName>
+        <fullName>stgHelpRelReciprocalMethod</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>stgHelpStoreErrorsOn</shortDescription>
-        <value>Check this if you&apos;d like errors to be stored in the database.</value>
+        <shortDescription>stgHelpRelReciprocalMethod</shortDescription>
+        <value>Determines method used to generate the reciprocal relationship. Either List Setting which uses custom settings by gender (see Reciprocal Relationships tab), or Value Inversion, which inverts the type if appropriate (&quot;Parent-Child&quot; to &quot;Child-Parent&quot;).</value>
     </labels>
     <labels>
-        <fullName>stgHelpRelAutoCreatedDup</fullName>
+        <fullName>stgHelpSimpleAddrChangeIsUpdate</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>stgHelpRelAutoCreatedDup</shortDescription>
-        <value>Auto-created Relationships are auto-deduplicated to prevent duplicate Relationship records from being created between two Contacts. Check this box to disable this behavior &amp; allow multiple Relationships of the same Type to be created between two Contacts</value>
+        <shortDescription>stgHelpSimpleAddrChangeIsUpdate</shortDescription>
+        <value>Treats changes to single Contact or Account address fields as updates to the existing address (i.e., no new Address object is created). Also causes white space and case changes to be treated as updates.</value>
     </labels>
     <labels>
-        <fullName>stgHelpRelReciprocalMethod</fullName>
+        <fullName>stgHelpStoreErrorsOn</fullName>
         <categories>Settings</categories>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>stgHelpRelReciprocalMethod</shortDescription>
-        <value>Determines method used to generate the reciprocal relationship. Either List Setting which uses custom settings by gender (see Reciprocal Relationships tab), or Value Inversion, which inverts the type if appropriate (&quot;Parent-Child&quot; to &quot;Child-Parent&quot;).</value>
+        <shortDescription>stgHelpStoreErrorsOn</shortDescription>
+        <value>Check this if you&apos;d like errors to be stored in the database.</value>
     </labels>
     <labels>
         <fullName>stgSendErrorsTitle</fullName>
```

Comment:
These are a lot of label changes. It's hard to review. The challenge here is that the ant updatePackageXml builds the labels file different from how the org builds it (the order if different). I don't mind having it one way or the other, but we need to make sure we don't lose any labels or the master build job will fail (labels cannot be removed from managed package).